test_runner/regress/test_import_pgdata.py

```diff
@@ -43,6 +48,22 @@ class RelBlockSize(Enum):
 ]
 
 
+def mock_import_bucket(vanilla_pg: VanillaPostgres, path: Path):
+    """
+    Mock the import S3 bucket into a local directory for a provided vanilla PG instance.
+    """
```

Comment:
would be nice to assert not vanilla_pg.running

Reply:
https://github.com/neondatabase/neon/pull/11614/commits/39eca768b31f75c592902496326c479c763ba1e8